apps/antalmanac/tests/patch-notes.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { describe, expect, it } from 'vitest';
```

Comment:
I prefer `test` instead of `it`.

```suggestion
import { describe, test, expect } from 'vitest';
```